### PR TITLE
Modify s2i repository to use Docker files generated by dogen

### PR DESCRIPTION
**Pull request by tmckayus**

### CODE REVIEW

File: Makefile.pyspark [hunk 1]
```diff
@@ -0,0 +1,24 @@
+LOCAL_IMAGE ?= radanalytics-pyspark
+
+# If you are going to push the built image to a registry
+# using the "push" make target then you should replace
+# "project" with an appropriate path for your registry and/or project
+PUSH_IMAGE=172.30.1.1:5000/myproject/radanalytics-pyspark
```
Comment (tmckayus) [inline Makefile.pyspark]: oops, this should come out

File: java-build/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+USER root
+RUN rm -rf /tmp/scripts
+USER root
+RUN rm -rf /tmp/artifacts
+
+USER 185
+
+
+CMD ["/usr/local/s2i/usage"]
```
[the remark this answers is not in the document]
Comment (tmckayus) [reply java-build/Dockerfile]: This is to allow the docker hub autobuilds to work.  The autobuilds will require the docker file, so we commit a build dir even though in a different context we may regenerate the docker file (and we'll regenerate the build dir whenever image yamls of scripts change).

File: image.java.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - name: "io.k8s.description"
+    value: "Platform for building a radanalytics java spark app"
+  - name: "io.k8s.display-name"
+    value: "radanalytics java_spark"
+  - name: "io.openshift.expose-services"
+    value: "8080:http"
+  - name: "io.openshift.s2i.scripts-url"
+    value: "image:///usr/local/s2i"
+  - name: "io.openshift.tags"
+    value: "builder,radanalytics,java_spark"
```
[the remark this answers is not in the document]
Comment (tmckayus) [reply image.java.yaml]: ack, good idea.

**Comment by crobby** [comment]
Build working well for me.  hook seems good.  Will do a few runs to explore more, but lgtm so far.

**Pull request merged**